controllers/etcdcopybackupstask_controller.go

```diff
@@ -369,7 +369,7 @@ func getConditionType(jobConditionType batchv1.JobConditionType) druidv1alpha1.C
 	return ""
 }
 
-func getJobName(task *druidv1alpha1.EtcdCopyBackupsTask) string {
+func getCopyJobName(task *druidv1alpha1.EtcdCopyBackupsTask) string {
```

Comment:
Let's use "copy backups" to be more concrete and stay aligned with other similar names.
```suggestion
func getCopyBackupsJobName(task *druidv1alpha1.EtcdCopyBackupsTask) string {
```

Reply:
Done